aries/aries_vcx_core/src/wallet/mod.rs

```diff
@@ -1,7 +1,11 @@
 pub mod agency_client_wallet;
+#[cfg(feature = "askar_wallet")]
+pub mod askar;
 pub mod base_wallet;
+pub mod constants;
 pub mod entry_tags;
 #[cfg(feature = "vdrtools_wallet")]
 pub mod indy;
 pub mod mock_wallet;
 pub mod structs_io;
+pub mod utils;
```

Comment:
Does not need to be public or exist (here) at all.

Reply:
This is no longer public.